<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_Kat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>651</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_Browse">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>38</y>
       <width>85</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz katalog</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Browse">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>581</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Maska">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_Maska">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>45</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Maska ogólna</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Maska">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_NazwPlik">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>821</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Definicja części nazw pliku</string>
    </property>
    <widget class="QLabel" name="label_Sep1">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Sep2">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Sep3">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Sep4">
     <property name="geometry">
      <rect>
       <x>517</x>
       <y>23</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Nazwa5">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>53</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Nazwa2">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>53</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Nazwa1">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>53</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Nazwa3">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>53</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Nazwa4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>53</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Reg1">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>92</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Reg2">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>92</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Reg3">
     <property name="geometry">
      <rect>
       <x>244</x>
       <y>92</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Reg4">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>92</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Reg5">
     <property name="geometry">
      <rect>
       <x>486</x>
       <y>92</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>125</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pomiń część przy szukaniu</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_CalaNazwa">
     <property name="geometry">
      <rect>
       <x>661</x>
       <y>21</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>21</y>
       <width>74</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Po całej nazwie</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>88</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reguły zdublowań</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cz4">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>24</y>
       <width>112</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cz2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>24</y>
       <width>112</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cz1">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cz5">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>24</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cz3">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>24</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <zorder>lineEdit_Cz4</zorder>
    <zorder>lineEdit_Cz2</zorder>
    <zorder>lineEdit_Cz1</zorder>
    <zorder>lineEdit_Cz5</zorder>
    <zorder>lineEdit_Cz3</zorder>
    <zorder>label_Sep1</zorder>
    <zorder>label_Sep2</zorder>
    <zorder>label_Sep3</zorder>
    <zorder>label_Sep4</zorder>
    <zorder>checkBox_Nazwa5</zorder>
    <zorder>checkBox_Nazwa2</zorder>
    <zorder>checkBox_Nazwa1</zorder>
    <zorder>checkBox_Nazwa3</zorder>
    <zorder>checkBox_Nazwa4</zorder>
    <zorder>comboBox_Reg1</zorder>
    <zorder>comboBox_Reg2</zorder>
    <zorder>comboBox_Reg3</zorder>
    <zorder>comboBox_Reg4</zorder>
    <zorder>comboBox_Reg5</zorder>
    <zorder>label_3</zorder>
    <zorder>checkBox_CalaNazwa</zorder>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton_Analyse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MIME">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check MIME Type</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Zdubl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Bledy">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>191</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Test">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>165</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Filter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFilter"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Total">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox_Kat</zorder>
   <zorder>groupBox_Maska</zorder>
   <zorder>groupBox_NazwPlik</zorder>
   <zorder>pushButton_Analyse</zorder>
   <zorder>pushButton_MIME</zorder>
   <zorder>listWidget_Zdubl</zorder>
   <zorder>listWidget_Bledy</zorder>
   <zorder>label_Test</zorder>
   <zorder>label_Filter</zorder>
   <zorder>lineEditFilter</zorder>
   <zorder>label_4</zorder>
   <zorder>label_Total</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
